web_src/js/features/common-button.ts

```diff
@@ -62,12 +65,15 @@ export function initGlobalDeleteButton(): void {
               postData.append('id', value);
             }
           }
-
-          const response = await POST(btn.getAttribute('data-url'), {data: postData});
-          if (response.ok) {
-            const data = await response.json();
-            window.location.href = data.redirect;
-          }
+          (async () => {
+            const response = await POST(btn.getAttribute('data-url'), {data: postData});
```

Comment:
Isn't there a `modal.classList.remove('is-loading')' missing?

Reply:
I think no because :

* when the request succeeds, the page gets reloaded
* when the request fails, there is no message feedback, so we can't make the form re-submit.
    * this part of code is for the legacy "delete-button", there are already many problems in it, so I think no need to do more.

![image](https://github.com/user-attachments/assets/d136a0f1-669f-455d-972a-773a60eea1b5)